lib/aws/generated/connect.ex

```diff
@@ -483,6 +760,52 @@ defmodule AWS.Connect do
     )
   end
 
+  @doc """
+  This API is in preview release for Amazon Connect and is subject to change.
+
+  Describes the specified queue.
+  """
+  def describe_queue(%Client{} = client, instance_id, queue_id, options \\ []) do
+    url_path = "/queues/#{URI.encode(instance_id)}/#{URI.encode(queue_id)}"
+    headers = []
+    query_params = []
+
+    Request.request_rest(
+      client,
+      metadata(),
+      :get,
+      url_path,
+      query_params,
+      headers,
+      nil,
+      options,
+      nil
+    )
+  end
+
+  @doc """
+  This API is in preview release for Amazon Connect and is subject to change.
```

Comment:
Interesting that they put preview APIs into the specs :)

Reply:
I was wondering if we should update only with latest tag instead of main branch?

Reply:
@vrcca it's a good point. They release a new version every week. We can aim to update only with the latest tag, yes :+1: